token_server/aws_get_credentials.js

```diff
@@ -0,0 +1,46 @@
+import AWS from 'aws-sdk';
+import fs from 'fs';
+import tmp from 'tmp';
+
+export {storeGoogleCredentials};
+
+const AWS_REGION = 'us-east-1';
+const SECRET_ID = 'arn:aws:secretsmanager:us-east-1:560508828482:secret:' +
```

Comment:
document or links to docs?

Reply:
Done, and realized I had left that out of docs, so important catch!